block/stream/output/state_changes.proto

```diff
@@ -41,8 +41,10 @@ option java_multiple_files = true;
 import "google/protobuf/wrappers.proto";
 import "basic_types.proto";
 import "exchange_rate.proto";
+import "state/addressbook/node.proto";
 import "state/blockrecords/block_info.proto";
 import "state/blockrecords/running_hashes.proto";
+import "state/blockstream/block_stream_info.proto";
```

Comment:
Is this in another PR?
I cannot find this file.

Reply:
Oops, didn't add! Should be there now.